src/core/js/adapt.js

```diff
@@ -106,9 +106,11 @@ define([
         }
 
         var navigationHeight = $(".navigation").outerHeight();
+        // prevent scroll issue when component description aria-label coincident with top of component
+        var ariaLabelBuffer = $(selector.is('.component')) ? 5 : 0;
```

Comment:
Present in IE* and Firefox. The above could work (amending to `.aria-label:first`), though I'd have to test the reported height (1px and 3px I think, respectively) would be sufficient.